modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -739,4 +739,80 @@ definition {
 		}
 	}
 
+	@description = "LPS-178783 - Verify a standalone action can be triggered manually to send an email"
+	@priority = 5
+	test CanSendEmailViaActionStandalone {
+		property custom.properties = "feature.flag.LPS-173894=true";
+		property portal.acceptance = "true";
+		property test.name.skip.portal.instance = "ObjectNotifications.CanSendEmailViaActionStandalone";
+		property test.smtp.server.enabled = "true";
+
+		task ("Given an Email Notification Template") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				emailBody = "Email Body",
+				entryTo = "test@liferay.com",
+				fromAddress = "test@liferay.com",
+				fromName = "Test Test",
+				name = "Notification Template",
+				notificationType = "Email",
+				subject = "Subject");
+
+			PortletEntry.save();
+		}
+
+		task ("And Custom Object with an Action (Standalone)") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 178783",
+				objectName = "CustomObject178783",
+				pluralLabelName = "Custom Objects 178783");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject178783");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject178783");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 178783");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionLabel = "Notification action when entry is added",
+				active = "true",
+				inputExpression = '''isEmpty(oldValue("customField"))''',
+				notificationName = "Notification Template",
+				objectLabel = "Custom Object 178783",
+				thenAction = "Notification",
+				whenAction = "On After Add");
```

Comment:
You need to create a standalone action here. You can find examples in ObjectActions.testcase